doc/nrf/gs_ins_windows.rst

```diff
@@ -46,113 +52,190 @@ To set up the toolchain, complete the following steps:
+        Therefore, you cannot use it to check out v0.1.0 or v0.3.0.
+   * To check out the latest state of development, enter the following command:
 
-      git clone https://github.com/NordicPlayground/nrfxlib.git nrfxlib
+     .. parsed-literal::
+        :class: highlight
 
-.. dirstructure_start
+        west init -m https\://github.com/NordicPlayground/fw-nrfconnect-nrf
+
+   This will clone the manifest repository (``nrf``).
+#. Enter the following command to clone the project repositories::
+
+      west update
 
 Your directory structure now looks like this::
 
    ncs
+    |___ .west
     |___ mcuboot
     |___ nrf
     |___ nrfxlib
     |___ zephyr
 
-.. dirstructure_end
 
-The latest state of development is on the master branches of the repositories.
-Note that this might not be a stable state.
-Therefore, unless you are familiar with the development process, you should always work with a specific release of the |NCS|.
+Updating the repositories
+=========================
+
+If you work with a specific release of the |NCS|, you do not need to update your repositories, because the release will not change.
+However, you might want to switch to a newer release or check out the latest state of development.
+
+To manage the ``nrf`` repository (the manifest repository), use git.
+Checking out a branch or tag in the ``nrf`` repository gives you a different version of the manifest file.
+Running ``west update`` will then update the project repositories to the state specified in this manifest file.
+
+For example, to switch to release v0.5.0 of the |NCS|, enter the following commands in the ``ncs/nrf`` directory::
 
-To do so, go to the :ref:`release_notes` of the latest release and check which tags to use for each of the repositories.
-Then go to each repository in turn and check out the respective tag.
+   git checkout v0.5.0
+   west update
 
-For example, to check out the v0.3.0 tag for the nrf repository, you would enter the following commands::
+To switch to the latest state of development, enter the following commands::
 
-   cd nrf
-   git checkout tags/v0.3.0
-   cd ..
+   git checkout master
+   west update
+
+
+Updating your existing clones to use West
+=========================================
+
+If you already cloned the |NCS| repositories in git and want to continue using these clones instead of creating new ones, you can initialize West to use your clones.
+All branches, remotes, and other configuration in your repositories will be maintained.
+
+To update your repositories to be managed by West, make sure that they are structured and named in the following way::
+
+   ncs
+    |___ mcuboot
+    |___ nrf
+    |___ nrfxlib
+    |___ zephyr
+
+Then complete the following steps:
+
+1. Open a |bash| in the ``ncs`` folder.
+#. Initialize West with the manifest folder from the current branch of your ``nrf`` repository::
+
+      west init -l nrf
+
+   This will update the manifest repository (``nrf``).
```

Comment:
This is incorrect, needs fixing: This will not touch **in any way** the manifest repo (nrf), it will only create the `.west/` folder with all that's required to use west